### ISSUE TITLE
Building 0.3.0 on Debian Linux

### ISSUE THREAD

**Issue by canonex** [AlexandrePTJ/kemai#10]
I'm trying to compile Kemai 0.3.0 on Debian linux and I get:

```
$ cmake  . -B build -DCMAKE_BUILD_TYPE=Release
-- The CXX compiler identification is GNU 8.3.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:13 (FetchContent_MakeAvailable):
  Unknown CMake command "FetchContent_MakeAvailable".


-- Configuring incomplete, errors occurred!
See also "CMakeOutput.log".
$ cmake . -B build -DCMAKE_BUILD_TYPE=Release
CMake Error at CMakeLists.txt:13 (FetchContent_MakeAvailable):
  Unknown CMake command "FetchContent_MakeAvailable".


-- Configuring incomplete, errors occurred!
See also "CMakeOutput.log".
```
Attached you find the CMakeOutput.log.
[CMakeOutput.log](https://github.com/AlexandrePTJ/kemai/files/4763595/CMakeOutput.log)

Thank you
Riccardo

**Comment by AlexandrePTJ** [comment]
Minimal version of CMake is 3.14.
It looks like debian version is older. I will look on how to be compatible with older version of CMake.

**Comment by AlexandrePTJ** [comment]
Fixed in 0.3.1

**Comment by canonex** [comment]
Wow, thank you!
